### PR TITLE
Image uploads to use DO

### PR DESCRIPTION
**Pull request by rideam**
- run the `migrations/20220904203809-add-harvest-image-url.js` to add the `harvest_image_url` column 
- uploaded image url saved to `harvest_image_url`
- new route `/migrateimages` in `routes/harvest.js` to migrate already saved images 
- new route `/getimagehash` in `routes/harvest.js` to fetch image from DO and compute the sha256 hash (`base64`) before adding the harvest entry data to blockchain

### CODE REVIEW

**Comment by jajukajulz** [comment]
Looking good. 

User model may need user_image_url column and auth.js - router.post('/register/whatsapp', async function (req, res) updating - we get a URL from Twilio with the image - wondering whether we need to then download the image and upload to DO instead of simply saving the Twilio URL?

And then the foodprint_weeklyview model also need harvest_photo_url,  and then the route i.e. qrcode.js - router.get('/scan/:id', and template scanresult.ejs need to be updated as well. The weekly view is updated when you insert a record into the Storage logbook (router.post('save'

File: views/harvestlogbook.ejs [hunk 1]
```diff
@@ -107,7 +107,7 @@
...
                                                     data-harvest_suppliername="<%=data[i].harvest_supplierName%>" data-harvest_farmername="<%=data[i].harvest_farmerName%>" data-harvest_supplieraddress="<%=data[i].harvest_supplierAddress%>" data-harvest_producename="<%=data[i].harvest_produceName%>"
-                                                    data-harvest_photohash="<%=data[i].harvest_photoHash%>" data-harvest_timestamp="<%=data[i].harvest_TimeStamp%>" data-harvest_capturetime="<%=data[i].harvest_CaptureTime%>" data-harvest_description="<%=data[i].harvest_Description%>"
+                                                    data-harvest_photohash="<%=data[i].harvest_image_url%>" data-harvest_timestamp="<%=data[i].harvest_TimeStamp%>" data-harvest_capturetime="<%=data[i].harvest_CaptureTime%>" data-harvest_description="<%=data[i].harvest_Description%>"
```
Comment (jajukajulz) [inline views/harvestlogbook.ejs:110]: @rideam Have you tested that when you click on Add to Blockchain, the image and not the image_url is correctly hashed in the inline addToBlockchain JS function i.e. `                let photoHash = sha256(harvestPhotoHash);`?

The quick way to check is that if you do a SHA256 hash of the file url from the console, this should be different from the SHA256 that is stored in the blockchain record.

Comment (rideam) [reply views/harvestlogbook.ejs:110]: @jajukajulz with the above change the url and not the image is being hashed ,, trying to fetch the image before the call to the sha256 funtion is causing cors issues

Comment (rideam) [reply views/harvestlogbook.ejs:110]: this is fixed now ,, implemented a workaround to get the hash via the route `getimagehash` in harvest.js and not hash the file in ejs

**Pull request closed without merging**